### PR TITLE
Update code quality function spacing, bare except & long line changes

### PR DESCRIPTION
**Pull request by kdcis**
Updated function spacing
Added specific exception for bare except changes 
Updated long line changes

### CODE REVIEW

File: packages/collectd/pybitmessagestatus.py [hunk 1]
```diff
@@ -28,17 +30,19 @@ def config_callback(ObjConfiguration):
             apiInterface = node.values[0]
         elif key.lower() == "apiport" and node.values:
             apiPort = node.values[0]
-    pybmurl = "http://" + apiUsername + ":" + apiPassword + "@" + apiInterface+ ":" + str(int(apiPort)) + "/"
+    pybmurl = "http://" + apiUsername + ":" + apiPassword + "@" + apiInterface + ":" + str(int(apiPort)) + "/"
```
Comment (PeterSurda) [inline packages/collectd/pybitmessagestatus.py:33]: use `format`

File: packages/collectd/pybitmessagestatus.py [hunk 2]
```diff
@@ -28,17 +30,19 @@ def config_callback(ObjConfiguration):
...
     collectd.info('pybitmessagestatus.py config done')
 
+
 def read_callback():
     try:
         clientStatus = json.loads(api.clientStatus())
-    except:
+    except (json.decoder.JSONDecodeError, TypeError()):
```
Comment (PeterSurda) [inline packages/collectd/pybitmessagestatus.py:40]: careful about python2 backwards compatibility

Comment (PeterSurda) [inline packages/collectd/pybitmessagestatus.py:40]: also `api.clientStatus()` may raise a different exception

File: packages/collectd/pybitmessagestatus.py [hunk 3]
```diff
@@ -48,10 +52,11 @@ def read_callback():
         metric.type_instance = i.lower()
         try:
             metric.values = [clientStatus[i]]
-        except:
+        except (NameError, KeyError):
```
Comment (PeterSurda) [inline packages/collectd/pybitmessagestatus.py:55]: `TypeError` instead of `NameError`

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: packages/collectd/pybitmessagestatus.py (hunk 1 above)
Comment (kdcis) [reply packages/collectd/pybitmessagestatus.py]: Updated

File: packages/collectd/pybitmessagestatus.py (hunk 2 above)
Comment (kdcis) [reply packages/collectd/pybitmessagestatus.py]: Updated

**Review by PeterSurda — APPROVED** [review]

File: packages/collectd/pybitmessagestatus.py (hunk 2 above)
Comment (kdcis) [reply packages/collectd/pybitmessagestatus.py]: updated

File: packages/collectd/pybitmessagestatus.py (hunk 3 above)
Comment (kdcis) [reply packages/collectd/pybitmessagestatus.py]: updated

**Pull request merged**